tests/test_regression.py

```diff
@@ -27,6 +27,11 @@ def test_compare_images_two_plotters_same(sphere, tmpdir):
     with pytest.raises(TypeError):
         pv.compare_images(im1, pl1.ren_win)
 
+    # test that this fails when the plotter is closed
+    pl1.close()
+    with pytest.raises(AttributeError, match='already been closed'):
```

Comment:
Change this is the other end is changed
```suggestion
    with pytest.raises(RuntimeError, match='already been closed'):
```